build/yamls/antrea-aks.yml

```diff
@@ -3217,6 +3217,27 @@ data:
     # The Namespace where Antrea Multi-cluster Controller is running.
     # The default is antrea-agent's Namespace.
       namespace: ""
+
+    secondaryNetwork:
+      enable: false
+    # Secondary network OVS configuration
+      ovs:
```

Comment:
indentation seems wrong to me here

Reply:
Below is the secondaryNetwork specific dump from /etc/antrea/antrea-agent.conf (from the running antrea agent pod context. Indentation was inline with other attributes on the antrea-agent.conf).

just looking to see how we can ensure this is aligned? (like the go formatter). 
 
secondaryNetwork:
  enable: false
# Secondary network OVS configuration
  ovs:
    # Secondary network OVS integration bridge name. Ensure it doesn't conflict with your existing OpenVSwitch bridges.
    ovsIntegrationBridgeName: "br-secnet-int"
    # Secondary network OVS transport bridge name. Ensure it doesn't conflict with your existing OpenVSwitch bridges.
    ovsTransportBridgeName: "br-secnet-trans"
    # Datapath type to use for the OpenVSwitch bridge created by Antrea. Supported values are:
    # - system
    # - netdev
    # 'system' is the default value and corresponds to the kernel datapath. Use 'netdev' to run
    # OVS in userspace mode. Userspace mode requires the tun device driver to be available.
    ovsDatapathType: "system"
    # Name of the OVS patch port which connects the integration and transport bridge.
    ovsPatchPort: "br-secnet-patch0"
  # Tunnel protocol used for encapsulating traffic across Nodes. It must be one
  # of "geneve", "vxlan", "gre", "stt".
  tunnelType: "geneve"